libs/shared/sentry/src/lib/utils/beforeSend.server.ts

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { tagFxaName } from './tagFxaName';
+import { InitSentryOpts } from '../models/SentryConfigOpts';
+import { ErrorEvent } from '@sentry/core';
+
+const IGNORE = new Set([
```

Comment:
Best thing here would be to have the variable name describe why we're ignoring it, or to leave a comment to that effect
```suggestion
const EXPECTED_ERRORS = new Set([
```